OpenOversight/app/main/views.py

```diff
@@ -670,15 +669,24 @@ def label_data(department_id=None, image_id=None):
         officer_exists = Officer.query.filter_by(id=form.officer_id.data).first()
         existing_tag = db.session.query(Face) \
                          .filter(Face.officer_id == form.officer_id.data) \
-                         .filter(Face.img_id == form.image_id.data).first()
+                         .filter(Face.original_image_id == form.image_id.data).first()
         if not officer_exists:
             flash('Invalid officer ID. Please select a valid OpenOversight ID!')
         elif not existing_tag:
             left = form.dataX.data
             upper = form.dataY.data
             right = left + form.dataWidth.data
             lower = upper + form.dataHeight.data
-            cropped_image = get_uploaded_cropped_image(image, (left, upper, right, lower))
+            byte_io = BytesIO()
+
+            if 'http' or 'https' in proper_path:
```

Comment:
I believe this should be `if 'http' in proper_path or 'https' in proper_path:` otherwise it always evals as true

Reply:
Very true!  Thank you for catching this syntax error.